misk-jooq/src/test/generated/kotlin/app/cash/backfila/client/misk/jooq/gen/DefaultCatalog.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * This file is generated by jOOQ.
+ */
+package app.cash.backfila.client.misk.jooq.gen
```

Comment:
Do we need the generated files checked in?

Reply:
Yes. It's easier if they are. We are generating these files by starting a docker container with mysql and then running migrations on it. This could be done in a CI. It takes time to run though. Easier to check it in.